### PR TITLE
Remove matrix for Deno template

### PR DESCRIPTION
**Pull request by dsherret**
Based on #1006, this removes the matrix to simplify the template and fix the deno fmt issue on Windows with actions/checkout as outlined in #1006.

> The actions/checkout action on windows checks out files with \r\n line endings instead of preserving them. This causes deno fmt --check to fail because the line endings are \r\n instead of \n.

### CODE REVIEW

File: ci/deno.yml [hunk 1]
```diff
@@ -16,22 +16,17 @@ on:
...
+    runs-on: ubuntu-latest
 
     steps:
       - name: Setup repo
         uses: actions/checkout@v2
 
       - name: Setup Deno
         # uses: denoland/setup-deno@v1
-        uses: denoland/setup-deno@4a4e59637fa62bd6c086a216c7e4c5b457ea9e79
+        uses: denoland/setup-deno@004814556e37c54a2f6e31384c9e18e9833173669
```
Comment (dsherret) [inline ci/deno.yml:27]: Not sure why this is done, but ugraded to https://github.com/denoland/setup-deno/releases/tag/v1.1.0

File: ci/deno.yml [hunk 2]
```diff
@@ -16,22 +16,17 @@ on:
 
 jobs:
   test:
-    runs-on: ${{ matrix.os }} # runs a test on Ubuntu, Windows and macOS
-
-    strategy:
-      matrix:
-        deno: ["v1.x", "canary"]
```
Comment (joshmgross) [inline ci/deno.yml:23]: Should we keep the deno version matrix and only remove the OS matrix?

Comment (justinmchase) [reply ci/deno.yml:23]: This is only a quick start too, people can adapt their own pipelines based on their needs after getting kick started with this. I agree its nice to have the simplest, fasted and most stable setup for the initial pipeline template.

**Review by justinmchase — APPROVED** [review]

**Review by joshmgross — APPROVED** [review]
LGTM, thanks!

**Comment by Vut11** [comment]
ช่วยกู้เงินได้หน่อย

**Comment by Vut11** [comment]
ช่วยกู้เงินได้หน่อย

**Pull request merged**